src/main/resources/ctp/types/payment-paypal.json

```diff
@@ -0,0 +1,215 @@
+{
+  "key": "payment-paypal",
+  "name": {
+    "en": "payment-paypal"
+  },
+  "resourceTypeIds": [
+    "payment"
+  ],
+  "fieldDefinitions": [
+    {
+      "name": "languageCode",
+      "label": {
+        "en": "languageCode"
+      },
+      "required": true,
+      "type": {
+        "name": "String"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "reference",
+      "label": {
+        "en": "reference"
+      },
+      "required": true,
+      "type": {
+        "name": "String"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "referenceText",
+      "label": {
+        "en": "referenceText"
+      },
+      "required": false,
+      "type": {
+        "name": "String"
+      },
+      "inputHint": "MultiLine"
+    },
+    {
+      "name": "redirectUrl",
+      "label": {
+        "en": "redirectUrl"
+      },
+      "required": false,
+      "type": {
+        "name": "String"
+      },
+      "inputHint": "SingleLine"
+    },
+    {
+      "name": "successUrl",
+      "label": {
+        "en": "successUrl"
+      },
+      "required": false,
```

Comment:
Is it not required? https://github.com/commercetools/commercetools-paypal-plus-integration/pull/109/files#diff-2cd7e972a9ecd144aeb5128d052e7e97R55

Reply:
Thanks!
92042f1450ae24664a9d2e2829784050560aaf7a